.github/workflows/main.yml

```
@@ -70,6 +70,9 @@ jobs:
     name: Publish release
     permissions:
       contents: write
+      id-token: write # This is required for requesting the JWT
     uses: ./.github/workflows/publish-release.yml
     secrets:
-      METAMASKBOT_TOKEN: ${{ secrets.METAMASKBOT_TOKEN }}
+      AWS_REGION: ${{ secrets.AWS_REGION }}
```

Is the region supposed to be hidden as well? Secrets, once set, are not readable, which may increase difficulty of debugging.
```suggestion
      AWS_REGION: ${{ vars.AWS_REGION }}
```

---

updated as well as the others